Formula/o/opencsg.rb

```diff
@@ -18,17 +18,16 @@ class Opencsg < Formula
     sha256 cellar: :any_skip_relocation, x86_64_linux:  "b37ea3b9c6749f57cf8b91102f334a161b707a27e9c5d4f939ea946b445207e0"
   end
 
-  depends_on "qt" => :build
+  depends_on "cmake" => :build
   depends_on "glew"
 
   def install
-    # Disable building examples
-    inreplace "opencsg.pro", "src example", "src"
+    # Target "opencsg" links to: OpenGL::OpenGL but the target was not found.
```

Comment:
this looks like an upstream issue, can you PR to [here](https://github.com/floriankirsch/OpenCSG/blob/2762e5e9d84dbf28ed4315ceb406ccbeece0198b/src/CMakeLists.txt#L38)?